### PR TITLE
refactor(colors): replace legacy colors by tokens

### PR DESCRIPTION
**Pull request by AxelPeter**
All legacy colors has been replaced by the corresponding ODS tokens.
Except for `@oui-color-primary`, `@oui-color-secondary`, `@oui-color-tertiary` and `@oui-color-brand`.

### CODE REVIEW

File: src/less/bootstrap/pagination.less [hunk 1]
```diff
@@ -91,9 +91,9 @@
     //   &,
     //   &:hover,
     //   &:focus {
-    //     color: @oui-color-silver;
-    //     border-color: @oui-color-concrete;
-    //     background-color: @oui-color-concrete;
+    //     color: @g-300;
```
Comment (antleblanc) [inline src/less/bootstrap/pagination.less:94]: Is this part couldn't be removed? (commented code)

Comment (AxelPeter) [reply src/less/bootstrap/pagination.less:94]: Good catch, I'll remove those parts

**Comment by AxelPeter** [comment]
To view the changes in the demo page, we need to rebuild the `assets` files.
With this project, we don't bother with any `grunt` or `webpack` config (since it was supposed to be temporary :) )

**Comment by antleblanc** [comment]
> To view the changes in the demo page, we need to rebuild the `assets` files.
> With this project, we don't bother with any `grunt` or `webpack` config (since it was supposed to be temporary :) )

Thank you for the information.

Maybe we can consider to stop versioning the `*.map` files into the `docs` folder.
Here is an attempt to address it: #95 

Please let me know what do you think.

Thanks

**Comment by AxelPeter** [comment]
I don't agree, because the `.map` file works with the versioned CSS file. If we rebuild the LESS files, the references of the map file changes according to its source files. So it needs to be versioned with the CSS file.

It has its place in the `docs` folder because the demo page is used as a dev environment.

**Comment by antleblanc** [comment]
> I don't agree, because the `.map` file works with the versioned CSS file. If we rebuild the LESS files, the references of the map file changes according to its source files. So it needs to be versioned with the CSS file.
> 
> It has its place in the `docs` folder because the demo page is used as a dev environment.

I'm OK with the purpose of the `*.map` files here and generally but it could be questioned for this project, especially for the `docs` folder.

We can keep it as is for the scope of this Pull Request in order to be able to merge it and then continue the disccussion here #95

Thanks

**Pull request merged**